jenkins/cut-release

```diff
@@ -1,13 +1,18 @@
 #!/bin/bash
-# Parameters:
+
+_required_variables='WHICH_REPO REPO_BASE'
```

Comment:
These normally get set via Jenkins, but we enforce their existence here in case we're running the script locally during development.